### ISSUE TITLE
This is a real bad egg or just a joke?

### ISSUE THREAD

**Issue by Vinisanic** [BuffelSaft/pokeemerald#278]
![imagem_2021-11-02_191504](https://user-images.githubusercontent.com/93102268/139959208-bb741b81-b86e-417d-9f32-cd9737e57fd9.png)

**Comment by BuffelSaft** [comment]
Probably a real bad egg, currently trying to find out what causes it.

**Comment by Vinisanic** [comment]
> Probably a real bad egg, currently trying to find out what causes it.

I'm scared of losing my save because of this bad egg

**Comment by BuffelSaft** [comment]
Bad Eggs can't destroy a save, they're not a virus. That's just one of those "mew under the truck" rumours

**Comment by Vinisanic** [comment]
> Bad Eggs can't destroy a save, they're not a virus. That's just one of those "mew under the truck" rumours

My Lucario turned into a bad egg

**Comment by Vinisanic** [comment]
> Bad Eggs can't destroy a save, they're not a virus. That's just one of those "mew under the truck" rumours

Ok, thanks

**Comment by null1024** [comment]
I was just checking in on the PC and had a Bad Egg show up in place of a Claydol I'd caught a few hours ago. I don't know if it ever made it to the PC properly, I hadn't checked until now.
Here's a screenshot and my save, if that helps.
![Pokemon Inclement Emerald-13](https://user-images.githubusercontent.com/1013451/142552721-17202888-215f-45d3-a090-c6049e55591e.png)
[badegg-inclement-emerald.zip](https://github.com/BuffelSaft/pokeemerald/files/7567312/badegg-inclement-emerald.zip)

As an aside, is there a way to remove Bad Eggs from the box?

**Comment by kleeenexfeu** [comment]
> I was just checking in on the PC and had a Bad Egg show up in place of a Claydol I'd caught a few hours ago. I don't know if it ever made it to the PC properly, I hadn't checked until now. Here's a screenshot and my save, if that helps. ![Pokemon Inclement Emerald-13](https://user-images.githubusercontent.com/1013451/142552721-17202888-215f-45d3-a090-c6049e55591e.png) [badegg-inclement-emerald.zip](https://github.com/BuffelSaft/pokeemerald/files/7567312/badegg-inclement-emerald.zip)
> 
> As an aside, is there a way to remove Bad Eggs from the box?

What version was it ? Is it from an old save? Can you describe what you did between the last time you checked your PC without bad eggs and with bad eggs ?
More recent version should fix this issue (although it's always possible that more code corrupt the RAM)

**Comment by null1024** [comment]
The time period for when I probably got the bad egg is hazy for me and I'm unfortunately not sure. Given the listed Pokemon there (I'd been catching every Pokemon I saw at the time), I entered the Sandstrewn Ruins from the Mirage Tower, went back to Mauville to heal, and that save has exactly when I checked the box.

I started the file on version 1.10.2, so I might have actually been unlucky enough to download the game a bit before the fix, I grabbed it on the 16th.
I've updated to 1.11 now.
